POC.md

```diff
@@ -0,0 +1,51 @@
+# Proof-of-Concept
+
+We will be implementing PoC that show people how to customize with Web Chat. Here is the list:
+
+## P0
+- Command-and-observe using Redux actions [#1147](https://github.com/Microsoft/BotFramework-WebChat/issues/1147)
+- Custom activities and attachments
+   - GitHub stargazers [#1148](https://github.com/Microsoft/BotFramework-WebChat/issues/1148)
+- Decorators
+   - Upvote/downvote button next to the activity [#1149](https://github.com/Microsoft/BotFramework-WebChat/issues/1149)
+   - Highlight activities [#1150](https://github.com/Microsoft/BotFramework-WebChat/issues/1150)
+- Disable interactivity [#1151](https://github.com/Microsoft/BotFramework-WebChat/issues/1151)
+- Handsfree mode [#1152](https://github.com/Microsoft/BotFramework-WebChat/issues/1152)
+- Resume conversation with past history
+- Unbundle Adaptive Cards [#1153](https://github.com/Microsoft/BotFramework-WebChat/issues/1153)
+
+## P1
+- API: Scroll activity into view
+- API: Set focus
+- Bring your own speech engine
+- Configuring collapsible timestamp
+- Customizing bot avatar
+- Handshake auth token
+- Minimize/hide Web Chat
+- Ponyfill for `window.open`
+- Typeahead/auto-complete (persistent menu)
+- Welcome messages
+- XML text format
+
+## P2
+- Always on/off microphone
+- Auto-scroll on incoming message
```

Comment:
Not the default? I would expect an incoming message to scroll to view.

Reply:
It should be "Auto-scroll on outgoing message". The auto-scroll on incoming is default. 😜